revpi_core.c

```diff
@@ -365,9 +365,11 @@ static int pibridge_probe(struct platform_device *pdev)
 		goto err_release_fw;
 	}
 	/* run threads */
-	ret = set_kthread_prios(revpi_core_kthread_prios);
-	if (ret)
-		goto err_close_serial;
+	if (piDev_g.machine_type != REVPI_CONNECT_4) {
+		ret = set_kthread_prios(revpi_core_kthread_prios);
+		if (ret)
+			goto err_close_serial;
+	}
```

Comment:
We shouldn't be doing this on Core SE and Connect SE either.

If you add the `revpi_gate_supported` flag I suggested earlier and check for it here, you'll exclude all of them.

Reply:
See https://github.com/RevolutionPi/piControl/pull/94/commits/8103db794a9ce9b22e5d2e8001cf95a807b0d9ca

Reply:
Thanks. I note that `set_kthread_prios()` apparently doesn't break on the Core SE so far even though it should. That's because <a href="https://github.com/RevolutionPi/linux/blob/revpi-5.10/arch/arm/boot/dts/overlays/revpi-core-se-2022-overlay.dts">revpi-core-se-2022-overlay.dts</a> neglects to set the status of spi0 to disabled.

Could you open a follow-on task for that? Disabling spi0 may reduce energy consumption.

The same can't be done on the Connect SE because a Con CAN or other SPI-attached module might be connect to the right handside.

Reply:
See https://github.com/RevolutionPi/linux/pull/156